### ISSUE TITLE
The console does not handle multiline strings

### ISSUE THREAD

**Issue by videah** [Catlinman/LOVEConsole#7]
Printing multiline strings to the console causes text to overlap and and has the input line placed one after where a single line string would be, rather than at the end of the multiline string.

**Comment by zealsprince** [comment]
Yes, thinking of splitting strings and pushing them to a new line on the stack seems to be a good thing to do. Same goes for strings that are too long to display on a single line.

I'll look into this.
